client/lib/Hero.ts

```diff
@@ -226,16 +225,26 @@ export default class Hero extends AwaitedEventTarget<{
 
   // METHODS
 
-  public async detach(elementOrResource: any, options: IDetachElementOptions = {}): Promise<void> {
+  public async addToDetached(name: string, elementOrResource: any): Promise<void> {
```

Comment:
We should probably type this as a Resource of a ISuperElement. I'm not a fan of leaving "any" in an API. I must have missed this last time.

Reply:
Fixed